Userland/Libraries/LibWeb/WebAudio/BiquadFilterNode.cpp

```diff
@@ -22,15 +22,15 @@ BiquadFilterNode::BiquadFilterNode(JS::Realm& realm, JS::NonnullGCPtr<BaseAudioC
     , m_q(AudioParam::create(realm, options.q, NumericLimits<float>::lowest(), NumericLimits<float>::max(), Bindings::AutomationRate::ARate))
     , m_gain(AudioParam::create(realm, options.gain, NumericLimits<float>::lowest(), NumericLimits<float>::max(), Bindings::AutomationRate::ARate))
 {
+    set_type(options.type);
```

Comment:
Ideally in the constructor initializer just like everything else (i.e)

m_type(options.type)

otherwise LGTM

Reply:
I used to think accessing setter is always preferable over writing a private variable directly, but I don't really care as long as it works